stories/topics/proc-aap-aws-backup-and-recovery.adoc

```diff
@@ -0,0 +1,68 @@
+[id="proc-aap-aws-backup-and-recovery"]
+
+In order to backup and restore your AAP deployment, it is vital to ensure automatic backups have been set for EFS and that these backups are accessible for restoration. Upon creating an AAP deployment, automatic EFS backups are set by default, however it is still prudent to check if those backups can be restored before a disaster scenario. In addition to this, it is also extremely important that regular manual snapshots of the RDS are taken to ensure a deployment can be restored as close as possible to its previous working state. Follow the steps below to ensure a smooth backup and recovery process.
+
+== Backup Process
+
+=== Take a manual snapshot of the RDS for your AAP deployment
+. Navigate to *CloudFormation* on your AWS Console
+. Select the *Resources* tab and navigate to menu:rds[rds] on the resource tree and click the link to go to the RDS of the deployment
+. Select *Instance actions* near the top of the screen to reveal a drop-down menu and click the btn:[Take Snapshot] button
+. Enter a name for the snapshot and click btn:[Take Snapshot]
+. Your snapshot should now appear in a list of manual RDS snapshots. It is important to take a manual snapshot over relying on the automated ones created by an instance as automated snapshots are attached to the lifecycle of the RDS instance. An automated snapshot will not be available to be used for recovery in the event your deployment is lost
```

Comment:
You cannot use an automated snapshot for recovery in the event that your deployment is lost.